pkgs/development/tools/analysis/qcachegrind/default.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv, fetchurl, bash, cmake, qt57, graphviz,  qmakeHook, makeQtWrapper, makeWrapper }:
+
+stdenv.mkDerivation rec {
+  name = "kcachegrind-${version}";
+  version = "16.12.2";
+
+  src = fetchurl {
+    url = "http://download.kde.org/stable/applications/${version}/src/${name}.tar.xz";
```

Comment:
I wonder if qcachegrind should be a part of kdeApplications since it uses its sources.